breakpad-symbols/src/http.rs

```diff
@@ -470,8 +537,42 @@ impl SymbolSupplier for HttpSymbolSupplier {
         &self,
         module: &(dyn Module + Sync),
     ) -> Result<SymbolFile, SymbolError> {
+        // If we don't have a debug_file or debug_identifier, then try to get it
+        // from a symbol server.
+        let mut debug_file = module.debug_file().unwrap_or_default().into_owned();
+        let mut debug_id = module.debug_identifier().unwrap_or_default();
```

Comment:
`SimpleModule` fields are all `Option<>` objects, so you could keep `debug_file` and `debug_id` options too, like:
```rust
        let mut debug_file = module.debug_file().map(|name| name.into_owned());
        let mut debug_id = module.debug_identifier();
```